### DISCUSSION TITLE
Is there any way of using elements of the webpage as embedded frames ?

### DISCUSSION THREAD

**Discussion by Allan-Hoejberg** [UtilitechAS/amsreader-firmware#379] [General]
I would like to display the Future prices on my Home assistant dashboard, is there any direct url where I can get that frame ?

**Comment by gskjold** [comment]
There is currently no way to include graphical elements directly. You can get the data to HA through MQTT. Prices will be published to MQTT once every hour (when price for current hour change)

**Comment by Nettopp** [comment]
It would be nice to be able to link to the different graphic elements as images/url. I am exporting data to Homeseer by MQTT, but there are limited options in Homeseer/HSTouch to display graphical elements of the collected data.
